MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -93,7 +93,7 @@ open class BaseInstructionsBannerView: UIControl {
     /**
      Updates the instructions banner info with a given `VisualInstructionBanner`.
      */
-    public func updateInstruction(_ instruction: VisualInstructionBanner?) {
+    public func update(for instruction: VisualInstructionBanner?) {
```

Comment:
Since this method is public, we should mention in the changelog that it has been renamed. Also, it appears to be inaccessible to Objective-C.